### PR TITLE
icons: fix icons not appearing

### PR DESCRIPTION
**Pull request by NRodriguezcuellar**
partly closes https://github.com/inveniosoftware/invenio-app-rdm/issues/1742

Screenshot from 2022-06-16 11-39-24
Screenshot from 2022-06-16 09-46-57
Screenshot from 2022-06-16 09-46-50

### CODE REVIEW

File: invenio_accounts/views/security.py [hunk 1]
```diff
@@ -32,8 +32,7 @@
     _(
         "%(icon)s Security",
         icon=make_lazy_string(
-            lambda: '<i class="{shield}".format(\
-            shield=current_theme_icons.shield)></i>'
+            lambda: '<i class="{shield}"></i>'.format(shield=current_theme_icons.shield)
```
[the remark this answers is not in the document]
Comment (NRodriguezcuellar) [reply invenio_accounts/views/security.py:35]: When you remove the lambda it returns an `out of context` error

**Comment by kpsherva** [comment]
sorry not sure if it was overlooked, or simply a wrong screnshot attached. In the first one there are still icons missing for `security` and `change password`? What about my dashboard and communities? do we assign icons to these items ?

**Comment by NRodriguezcuellar** [comment]
> sorry not sure if it was overlooked, or simply a wrong screnshot attached. In the first one there are still icons missing for `security` and `change password`? What about my dashboard and communities? do we assign icons to these items ?

sorry! outdated screenshot indeed, corrected it. About the other menu options i am not sure. I suggest to make another issue to discuss and implement adding icons to these menu items?

File: invenio_accounts/views/security.py (hunk 1 above)
Comment (kpsherva) [inline invenio_accounts/views/security.py:35]:
```suggestion
            lambda: '<i class="{icon}"></i>'.format(icon=current_theme_icons.shield)
```

**Pull request merged**